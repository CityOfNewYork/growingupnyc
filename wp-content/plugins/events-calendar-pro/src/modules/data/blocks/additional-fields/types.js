/**
 * Internal dependencies
 */
import { PREFIX_EVENTS_PRO_STORE } from '@moderntribe/events-pro/data/prefix';

export const SET_ADDITIONAL_FIELD_INITIAL_STATE = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_INITIAL_STATE`;
export const ADD_ADDITIONAL_FIELD = `${ PREFIX_EVENTS_PRO_STORE }/ADD_ADDITIONAL_FIELD`;
export const REMOVE_ADDITIONAL_FIELD = `${ PREFIX_EVENTS_PRO_STORE }/REMOVE_ADDITIONAL_FIELD`;

export const SET_ADDITIONAL_FIELD_LABEL = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_LABEL`;
export const SET_ADDITIONAL_FIELD_IS_PRISTINE = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_IS_PRISTINE`;
export const SET_ADDITIONAL_FIELD_VALUE = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_VALUE`;
export const SET_ADDITIONAL_FIELD_TYPE = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_TYPE`;
export const SET_ADDITIONAL_FIELD_OPTIONS = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_OPTIONS`;
export const SET_ADDITIONAL_FIELD_DIVIDER_LIST = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_DIVIDER_LIST`;
export const SET_ADDITIONAL_FIELD_DIVIDER_END = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_DIVIDER_END`;
export const APPEND_ADDITIONAL_FIELD_VALUE = `${ PREFIX_EVENTS_PRO_STORE }/APPEND_ADDITIONAL_FIELD_VALUE`;
export const REMOVE_ADDITIONAL_FIELD_VALUE = `${ PREFIX_EVENTS_PRO_STORE }/REMOVE_ADDITIONAL_FIELD_VALUE`;
export const SET_ADDITIONAL_FIELD_CHANGE = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_CHANGE`;
export const SET_ADDITIONAL_FIELD_META_KEY = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_META_KEY`;
export const SET_ADDITIONAL_FIELD_OUTPUT = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_OUTPUT`;
export const SET_ADDITIONAL_FIELD_BLUR = `${ PREFIX_EVENTS_PRO_STORE }/SET_ADDITIONAL_FIELD_BLUR`;
