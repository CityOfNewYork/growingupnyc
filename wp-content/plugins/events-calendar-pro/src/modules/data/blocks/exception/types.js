/* eslint-disable max-len */
/**
 * Internal dependencies
 */
import { PREFIX_EVENTS_PRO_STORE } from '@moderntribe/events-pro/data/prefix';

export const ADD_EXCEPTION_FIELD = `${ PREFIX_EVENTS_PRO_STORE }/ADD_EXCEPTION_FIELD`;
export const ADD_EXCEPTION = `${ PREFIX_EVENTS_PRO_STORE }/ADD_EXCEPTION`;
export const REMOVE_EXCEPTION_FIELD = `${ PREFIX_EVENTS_PRO_STORE }/REMOVE_EXCEPTION_FIELD`;
export const REMOVE_EXCEPTION = `${ PREFIX_EVENTS_PRO_STORE }/REMOVE_EXCEPTION`;
export const EDIT_EXCEPTION = `${ PREFIX_EVENTS_PRO_STORE }/EDIT_EXCEPTION`;
export const SYNC_EXCEPTIONS_FROM_DB = `${ PREFIX_EVENTS_PRO_STORE }/SYNC_EXCEPTIONS_FROM_DB`;
