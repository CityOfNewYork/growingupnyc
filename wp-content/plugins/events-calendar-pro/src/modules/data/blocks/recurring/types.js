/* eslint-disable max-len */
/**
 * Internal dependencies
 */
import { PREFIX_EVENTS_PRO_STORE } from '@moderntribe/events-pro/data/prefix';

export const ADD_RULE_FIELD = `${ PREFIX_EVENTS_PRO_STORE }/ADD_RULE_FIELD`;
export const ADD_RULE = `${ PREFIX_EVENTS_PRO_STORE }/ADD_RULE`;
export const REMOVE_RULE_FIELD = `${ PREFIX_EVENTS_PRO_STORE }/REMOVE_RULE_FIELD`;
export const REMOVE_RULE = `${ PREFIX_EVENTS_PRO_STORE }/REMOVE_RULE`;
export const EDIT_RULE = `${ PREFIX_EVENTS_PRO_STORE }/EDIT_RULE`;
export const SYNC_RULES_FROM_DB = `${ PREFIX_EVENTS_PRO_STORE }/SYNC_RULES_FROM_DB`;
