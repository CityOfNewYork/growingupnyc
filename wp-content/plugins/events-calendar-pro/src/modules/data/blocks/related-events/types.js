/**
 * Internal dependencies
 */
import { PREFIX_EVENTS_PRO_STORE } from '@moderntribe/events-pro/data/prefix';

export const SET_RELATED_EVENTS_INITIAL_STATE = `${ PREFIX_EVENTS_PRO_STORE }/SET_RELATED_EVENTS_INITIAL_STATE`;

export const SET_RELATED_EVENTS_TITLE = `${ PREFIX_EVENTS_PRO_STORE }/SET_RELATED_EVENTS_TITLE`;

export const SET_RELATED_EVENTS_DISPLAY_IMAGES = `${ PREFIX_EVENTS_PRO_STORE }/SET_RELATED_EVENTS_DISPLAY_IMAGES`;
