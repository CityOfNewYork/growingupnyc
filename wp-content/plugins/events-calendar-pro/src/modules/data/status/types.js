/* eslint-disable max-len */
/**
 * Internal dependencies
 */
import { PREFIX_EVENTS_PRO_STORE } from '@moderntribe/events-pro/data/prefix';

export const SET_SERIES_QUEUE_STATUS = `${ PREFIX_EVENTS_PRO_STORE }/SET_SERIES_QUEUE_STATUS`;
