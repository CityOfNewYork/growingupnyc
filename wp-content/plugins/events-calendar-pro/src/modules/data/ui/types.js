/* eslint-disable max-len */
/**
 * Internal dependencies
 */
import { PREFIX_EVENTS_PRO_STORE } from '@moderntribe/events-pro/data/prefix';

export const TOGGLE_REPEAT_EVENTS_BLOCK_VISIBILITY = `${ PREFIX_EVENTS_PRO_STORE }/TOGGLE_REPEAT_EVENTS_BLOCK_VISIBILITY`;

export const TOGGLE_RULE_PANEL_VISIBILITY = `${ PREFIX_EVENTS_PRO_STORE }/TOGGLE_RULE_PANEL_VISIBILITY`;
export const HIDE_RULE_PANEL = `${ PREFIX_EVENTS_PRO_STORE }/HIDE_RULE_PANEL`;

export const TOGGLE_RULE_PANEL_EXPAND = `${ PREFIX_EVENTS_PRO_STORE }/TOGGLE_RULE_PANEL_EXPAND`;
export const EXPAND_RULE_PANEL = `${ PREFIX_EVENTS_PRO_STORE }/EXPAND_RULE_PANEL`;

export const TOGGLE_EXCEPTION_PANEL_VISIBILITY = `${ PREFIX_EVENTS_PRO_STORE }/TOGGLE_EXCEPTION_PANEL_VISIBILITY`;
export const HIDE_EXCEPTION_PANEL = `${ PREFIX_EVENTS_PRO_STORE }/HIDE_EXCEPTION_PANEL`;

export const TOGGLE_EXCEPTION_PANEL_EXPAND = `${ PREFIX_EVENTS_PRO_STORE }/TOGGLE_EXCEPTION_PANEL_EXPAND`;
export const EXPAND_EXCEPTION_PANEL = `${ PREFIX_EVENTS_PRO_STORE }/EXPAND_EXCEPTION_PANEL`;
