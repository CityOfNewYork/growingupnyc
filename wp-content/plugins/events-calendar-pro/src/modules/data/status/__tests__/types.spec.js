/**
 * External Dependencies
 */
import * as types from '../types';

describe( 'Status types', () => {
	const keys = Object.keys( types );

	keys.forEach( ( key ) => {
		test( key, () => {
			expect( types[ key ] ).toMatchSnapshot();
		} );
	} );
} );
