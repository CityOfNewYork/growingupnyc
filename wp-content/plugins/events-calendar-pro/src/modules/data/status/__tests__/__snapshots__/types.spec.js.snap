// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Status types SET_SERIES_QUEUE_STATUS 1`] = `"@@MT/EVENTS-PRO/SET_SERIES_QUEUE_STATUS"`;
